TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift

```diff
@@ -20,17 +20,95 @@
 import Foundation
 
 struct PushFactor: Factor, Codable {
+
   var status: FactorStatus = .unverified
   let sid: String
   let friendlyName: String
   let accountSid: String
   let serviceSid: String
   let identity: String
-  let type: FactorType = .push
+  var type: FactorType = .push
+  var allowIphoneMigration: Bool = false
   let createdAt: Date
   let config: Config
   var keyPairAlias: String?
   var metadata: [String: String]?
+
+  init(
+    status: FactorStatus = .unverified,
+    sid: String,
+    friendlyName: String,
+    accountSid: String,
+    serviceSid: String,
+    identity: String,
+    type: FactorType = .push,
+    allowIphoneMigration: Bool = false,
+    createdAt: Date,
+    config: Config,
+    keyPairAlias: String? = nil,
+    metadata: [String : String]? = nil
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/eb579290ef458a5ac85d4e99b29e765308b08746/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/PushFactor.swift#L49"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>PushFactor.swift:49</code></td>
    </tr>
  </tbody>
</table>